tests/integration/ethernet_mtu_test.py

```diff
@@ -155,3 +156,14 @@ def test_set_mtu_on_two_vlans_with_a_shared_base(eth1_up):
         libnmstate.apply(desired_state)
 
         assertlib.assert_state(desired_state)
+
+
+@ip_monitor_assert_stable_link_up('eth1')
```

Comment:
I think you need an xfail over this one, as the monitor will pass only with ver >= 1.18.

Reply:
I think there is something wrong with `ip_monitor_assert_stable_link_up` the test passed in the CI.

Reply:
There is no need to use xfail here. Information related to this added on commit message.